web/src/features/moreCast2/components/GroupHeader.tsx

```diff
@@ -64,7 +64,7 @@ const GroupHeader = ({ id, columns, weatherParam, handleShowHideChange }: GroupH
         <Stack sx={{ overflow: 'hidden' }}>
           <PopoverHeader>Choose Models to Display</PopoverHeader>
           <FormGroup>
-            {columns.map(column => {
+            {columns?.map(column => {
```

Comment:
Looks like this isn't possibly undefined, do we need this change?

Reply:
The app works fine with the null check, but unit tests fail.

It looks like columns can be undefined when the app starts up. The culprit is:
`const [showHideColumnsModel, setShowHideColumnsModel] = useState<Record<string, ColumnVis[]>>({})`

We start with an empty object and down the chain we attempt to access the temp/rh/wind/precip properties of the empty object which don't yet exist which ends up with columns=undefined being passed in to the `GroupHeader` component. 

I might be able to fix it by initializing `showHideColumnsModel` in the `useState` hook.

Reply:
Yep, looks like initialization in the useState hook fixes the problem. I've reverted the null check.